editor/src/com/talosvfx/talos/editor/addons/scene/apps/tween/ScenePreviewStage.java

```diff
@@ -57,6 +57,11 @@ public void drawContent (Batch batch, float parentAlpha) {
         renderer.skipUpdates = !isPlaying;
     }
 
+    @Override
+    protected boolean canMoveAround () {
+        return isInViewPort || isDragging;
```

Comment:
maybe it's a bad idea to convert it to a function?
maybe just have the old variable and only update it in act?

Reply:
@Tom-Ski